insta/src/App.css

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
If, and only if you have extra time, try to add more styling to your components! :)